test/files/jvm/varargs-separate-bytecode/Props_1.flags

```diff
@@ -0,0 +1 @@
+-sourcepath test/files/jvm/varargs-separate-bytecode/
```

Comment:
Ok, it looks like appending _1 and _2 to source files makes them compile in separate compiler runs, so my last commit simplifies the test to not use `-sourcepath`. I can squash the commits later, for the moment I'll keep them as separate, so reviewers can track what's new.